lib/screens/alerts/alert.ex

```diff
@@ -456,9 +456,27 @@ defmodule Screens.Alerts.Alert do
     end
   end
 
-  def get_informed_platforms_from_entities(informed_entities, all_platforms_at_informed_stations) do
+  @spec get_informed_platforms_from_entities([InformedEntity.t()], [Stop.t()] | [Stop.id()]) :: [
+          InformedEntity.t()
+        ]
+  def get_informed_platforms_from_entities(
```

Comment:
For my own Elixir knowledge:

Is it preferred having multiple functions handling each of the cases rather than doing logic beforehand to map `Stop.t()` to `Stop.id()` and only using one function right?